README.md

```diff
@@ -67,11 +70,11 @@ const MuxVideo = muxReactNativeVideo(Video);
 ## Known Issues
 
 1. The `paused` property does not behave as expected on Android when using the default player controls. (the `onProgress` event, which is something the Mux SDK needs to hook into does not fire): https://github.com/react-native-community/react-native-video/issues/1979. If you are using the `paused` property it will work on iOS with the default controls but if you need to use it on Android you will have to implement your own controls and set/unset the `paused` property yourself.
+1. The `player_is_fullscreen` property is not reported as expected on iOS. The react-native-video callbacks for fullscreen status `onFullscreenPlayerDidPresent` and `onFullscreenPlayerDidDismiss` do not get called when entering fullscreen using the native controls: https://github.com/react-native-video/react-native-video/issues/552
 
 ## Caveats
 
-1. Upscale and downscale % metrics are not calculated because we are unable to get player width and player height measurments from react-native-video.
-1. Even if we could get player width and height, we still wouldn't be able to calculate upscale and downscale % metrics for HLS sources is because of this open issue related to getting the video source width & height: https://github.com/react-native-community/react-native-video/issues/1194.
+1. If we get player width and height, we still won't be able to calculate upscale and downscale % metrics for HLS sources because of this open issue related to getting the video source width & height: https://github.com/react-native-community/react-native-video/issues/1194.
```

Comment:
This bug got fixed, so on more recent versions of react-native-video we should be source width/height now. I'll verify

Reply:
This part didnt get fixed as far as I know https://github.com/react-native-video/react-native-video/issues/1194#issuecomment-596140286

Reply:
ohh, the part about rendition changes, I think you're right :/ . We should double-check and make sure we're not mis-reporting this number when renditions changes happen. Right now, we do actually track the reported naturalSize (if it is there).

https://github.com/muxinc/mux-stats-sdk-react-native-video/blob/master/src/index.js#L110-L115